--- Expansion.php  2019-04-16 18:58:13.254005966 +0000
+++ Expansion.php       2019-04-16 18:59:03.202020632 +0000
@@ -380,6 +380,9 @@
 
                     $delimiters = $result->{'delimiters'};
 
+                    // HACK ALERT: THIS SHOULDN'T HAPPEN AND SHOULD BE FIXED UPSTREAM
+                    if (!is_array($context)) $context = [$context];
+
                     // normalize associative arrays
                     if (array_values($context) !== $context) $context = [$context];
